projects/hslayers/src/components/compositions/compositions-catalogue.service.ts

```diff
@@ -74,7 +74,7 @@ export class HsCompositionsCatalogueService {
     private _zone: NgZone
   ) {
     this.hsCommonEndpointsService.endpointsFilled.subscribe((endpoints) => {
-      this.filteredEndpoints = this.getFilteredEndpointsForCompositions();
+      this.endpoints = endpoints;
```

Comment:
```suggestion
      this.endpoints = endpoints ?? [];
```

Better safe then sorry. `endpoints` could be `null` in case the `endpointsFilled` is never casted, since it's a BehaviorSubject initiated with a `null` value.